src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -92,14 +127,23 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of violation from the above config:
+ * Example:
  * </p>
  * <pre>
  * void method() {
- *   System.out. // violation
- *   print("Example");
+ *   System.out.print("Example"); // OK
```

Comment:
please use the same code as in previous example block to let user see how behavior is changing on same code by just config change.